desktop/src/main/java/bisq/desktop/components/paymentmethods/SwiftForm.java

```diff
@@ -0,0 +1,346 @@
+                    bankNameField.setText("");
+                    bankBranchField.setText("");
+                    bankAddressTextArea.setText("");
+                }
+                updateFromInputs();
+            });
+            // make the intermediary fields initially greyed out
+            for (Node x : gridPane2.getChildren()) {
+                if (x == useIntermediaryCheck)
+                    continue;
+                x.setDisable(!useIntermediaryCheck.isSelected());
+            }
+        }
+    }
+
+    private void addFieldsForBeneficiaryEdit() {
+        String label = Res.get("payment.account.owner");
+        InputTextField beneficiaryNameField = addInputTextField(gridPane, ++gridRow, label);
+        beneficiaryNameField.setPromptText(label);
+        beneficiaryNameField.setValidator(defaultValidator);
+        beneficiaryNameField.textProperty().addListener((ov, oldValue, newValue) -> {
+            formData.setBeneficiaryName(newValue.trim());
+            updateFromInputs();
+        });
+
+        label = Res.get(Res.get(SWIFT_ACCOUNT));
+        InputTextField beneficiaryAccountNrField = addInputTextField(gridPane, ++gridRow, label);
+        beneficiaryAccountNrField.setPromptText(label);
+        beneficiaryAccountNrField.setValidator(defaultValidator);
+        beneficiaryAccountNrField.setValidator(accountNrValidator);
+        beneficiaryAccountNrField.textProperty().addListener((ov, oldValue, newValue) -> {
+            formData.setBeneficiaryAccountNr(newValue.trim());
+            updateFromInputs();
+        });
+
+        label = Res.get(ADDRESS + BENEFICIARYPOSTFIX);
+        TextArea beneficiaryAddressTextArea = addTopLabelTextArea(gridPane, ++gridRow, label, label).second;
+        beneficiaryAddressTextArea.setMinHeight(70);
+        beneficiaryAddressTextArea.textProperty().addListener((ov, oldValue, newValue) -> {
+            formData.setBeneficiaryAddress(newValue.trim());
+            updateFromInputs();
+        });
+
+        label = Res.get(Res.get("payment.account.city"));
+        InputTextField beneficiaryCityField = addInputTextField(gridPane, ++gridRow, label);
+        beneficiaryCityField.setPromptText(label);
+        beneficiaryCityField.setValidator(defaultValidator);
+        beneficiaryCityField.textProperty().addListener((ov, oldValue, newValue) -> {
+            formData.setBeneficiaryCity(newValue.trim());
+            updateFromInputs();
+        });
+
+        label = Res.get(Res.get(PHONE + BENEFICIARYPOSTFIX));
+        InputTextField beneficiaryPhoneField = addInputTextField(gridPane, ++gridRow, label);
+        beneficiaryPhoneField.setPromptText(label);
+        beneficiaryPhoneField.setValidator(defaultValidator);
+        beneficiaryPhoneField.textProperty().addListener((ov, oldValue, newValue) -> {
+            formData.setBeneficiaryPhone(newValue.trim());
+            updateFromInputs();
+        });
+
+        label = Res.get("payment.shared.optionalExtraInfo");
```

Comment:
```suggestion
        label = Res.get("payment.shared.optionalExtra");
```
Wrong key
![Bildschirmfoto 2021-09-09 um 10 27 58](https://user-images.githubusercontent.com/170962/132651138-6b90dcde-42dd-4483-ab0c-0617c8c76f1a.png)